torch/storage.py

```diff
@@ -342,38 +342,63 @@ def float8_e4m3fnuz(self):
         """Casts this storage to float8_e4m3fnuz type"""
         return self._to(torch.float8_e4m3fnuz)
 
-    def is_pinned(self, device: Union[str, torch.device] = "cuda"):
+    def is_pinned(self, device: Union[None, str, torch.device] = None):
         r"""Determine whether the CPU storage is already pinned on device.
 
         Args:
-            device (str or torch.device): The device to pin memory on. Default: ``'cuda'``.
+            device (None, str or torch.device): The device to pin memory on. Default: ``None``.
+                This argument is deprecated, since the device is autoselected to pin memory on.
 
         Returns:
             A boolean variable.
         """
+        if device is not None:
+            warnings.warn(
```

Comment:
I'm not sure we actually want to remove this argument altogether. I think saying in the doc that using this argument is discouraged is enough in the short term?

Adding these warnings here is going to be very very spammy as all existing users of these APIs will now start to see them and we don't want that.

Reply:
Yes，too many warnings are annoying. But now, the default device of `storage.is_pinned/pin_memory` is 'cuda', not the current accelerator, is it acceptable?